modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -4920,6 +4920,35 @@ public void testGetObjectEntries() throws Exception {
 			).build());
 	}
 
+	@Test
```

Comment:
You need to include the `@FeatureFlag("LPD-53981")` annotation, otherwise CI will not enable FF Recycle Bin before running this test.